vignettes/decorate-module-output.Rmd

```diff
@@ -127,7 +133,7 @@ interactive_decorator <- teal_transform_module(
 )
 ```
 
-As in the earlier examples, `make_teal_transform_server()` can simplify the creation of the server component. 
+As in the earlier examples, `make_teal_transform_server()` can simplify the creation of the server component.
 This wrapper requires you to use `input` object names directly in the expression - note that we have `xlab(x_axis_table)` and not `my_title = input$x_axis_title` together with `xlab(my_title)`.
```

Comment:
```suggestion
This wrapper requires you to use `input` object names directly in the expression - note that we have `xlab(x_axis_title)` and not `my_title = input$x_axis_title` together with `xlab(my_title)`.
```
I'm a bit confused here, the explanation seems to be talking about `x_axis_title` and code for `y_axis_title`